### ISSUE TITLE
Raspberry Pi Sonos Controller with Spotify

### ISSUE THREAD

**Issue by Tobi2707** [janbar/noson-app#80]
I want to run a RPi al Sonos Controller with a Touch Screen mounted on the wall.
Is this possible with Noson App?
Is there full support for Spotify if I have a premium Spotify Account?
Would be nice if you can give me answers to this questions :)

**Comment by janbar** [comment]
Yep, it works great with Spotify and Deezer as well. You can try it with an android device, I provide the apk in release screen. Have a good fun.

**Comment by Tobi2707** [comment]
I have a raspberry pi here but no android device.
Do i need an android device?

**Comment by janbar** [comment]
No need android device. It just for trying if you need to check that works at home.
the app  runs on any Unix  with qt >= 5.9.
Also to have all functionalities you need pulse audio headers at compile time.

**Comment by Tobi2707** [comment]
Ahh and what do I need on Raspian? I will try it the next days :)  will be a good solution :)

**Comment by janbar** [comment]
The file debian/control from the sources contains all dependencies you need to achieve the build.
